### ISSUE TITLE
Linux Pip install fail.

### ISSUE THREAD

**Issue by MagazinnikIvan** [pywinauto/pywinauto#262]
On linux command `pip install pywinauto` fails by trying to install windows  module pywin32.
Fail example: [https://travis-ci.org/MagazinnikIvan/test_qt_app/builds/172109535](https://travis-ci.org/MagazinnikIvan/test_qt_app/builds/172109535)

**Comment by vasily-v-ryabov** [comment]
As a workaround this command should work
```
pip install https://github.com/pywinauto/pywinauto/releases/download/0.6.0/pywinauto-0.6.0.zip
```
@MagazinnikIvan could you please check?

**Comment by MagazinnikIvan** [comment]
@vasily-v-ryabov Yes this command work fine on both 2.7 and 3.5 python versions

**Comment by vasily-v-ryabov** [comment]
Please re-check again. I've updated zip file to make it working on Windows (`setup.cfg` (flake8 section) contained symbols that cannot be decoded).

**Comment by MagazinnikIvan** [comment]
@vasily-v-ryabov pip install from zip still work fine on 2.7 and 3.5 Pythons

**Comment by vasily-v-ryabov** [comment]
Updated the PyPI package. Please re-check `pip install pywinauto` now.

**Comment by MagazinnikIvan** [comment]
@vasily-v-ryabov Yes, now it works without errors on all pythons versions on linux.

**Comment by UnruffledST** [comment]
I am receiving this same error again back in lubuntu 19.10 ??

**Comment by liangyongxiang** [comment]
Me too. And I try this command:
```
pip install --user https://github.com/pywinauto/pywinauto/releases/download/0.6.8/pywinauto-0.6.8.tar.gz
```

> I am receiving this same error again back in lubuntu 19.10 ??
> 
> @vasily-v-ryabov

**Comment by airelil** [comment]
There's ongoing work in atspi branch to make pywinauto crosplatform. You can give it a try

**Comment by mo-han** [comment]
btw, on pypi.org, which old version is the latest cross-platform one?

**Comment by vasily-v-ryabov** [comment]
Hi @mo-han the latest PyPI version 0.6.8 is still not fully cross-platform. It has small function set on Linux (mouse and keyboard) and there is no macOS functions at all. You have to wait till next major release or try to manually install it from "atspi" branch:
```
pip install https://github.com/pywinauto/pywinauto/archive/atspi.zip
```

**Comment by mo-han** [comment]
@vasily-v-ryabov 
thx, and that's why i'd like to know the last OLD version which is cross-platform, do you know which one is?

**Comment by vasily-v-ryabov** [comment]
The whole 0.6.x series should be cross-platform. I mean it can be imported on Linux at least. For macOS you have to use conditional record in requirements.txt because it can't be imported on macOS.

If you can describe your problem, maybe I can suggest more ways to resolve it.

**Comment by mo-han** [comment]
@vasily-v-ryabov 
i'm pretty sure in "Ubuntu 18.04.4 LTS" the latest version 0.6.8 and the previous 0.6.7 both failed installation by `pip install pywinauto`
